src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingCTest.java

```diff
@@ -111,6 +111,11 @@
     @Deprecated(message = "Does nothing, because equals/hashcode don't always improve performance of verification")
     boolean requireStateEquivalenceImplCheck() default false;
 
+    /**
+     * Should the managed strategy use the static memory tracking and restoring algorithm.
+     */
+    boolean restoreStaticMemory() default DEFAULT_RESTORE_STATIC_MEMORY;
```

Comment:
Should we remove this parameter and always restore the memory?

Reply:
done